sqs-fifo-delayed-queue-dynamodb/delay_fifo_queue_test/delay_fifo_queue_test_stack.py

```diff
@@ -0,0 +1,108 @@
+    aws_cloudwatch as cloudwatch,
+    aws_cloudwatch_actions as cloudwatch_actions,
+    aws_dynamodb as dynamodb,
+    aws_sns as sns,
+    aws_sns_subscriptions as subscriptions
+)
+from constructs import Construct
+
+class DelayFifoQueueTestStack(Stack):
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)    
+        
+        #create a dead letter queue called primary_queue_dlq
+        primary_queue_dlq = sqs.Queue(self, "DelayFifoQueueDlq",
+                                      visibility_timeout=Duration.seconds(60),
+                                      fifo=True,
+                                      content_based_deduplication=True
+                                      )
+
+        # create an initial primary SQS FIFO queue with a visibility timeout of 60 seconds
+        primary_queue = sqs.Queue(self, "DelayFifoQueue",
+                          visibility_timeout=Duration.seconds(60),
+                          fifo=True,
+                          content_based_deduplication=True,
+                          dead_letter_queue=sqs.DeadLetterQueue(
+                            max_receive_count=5,
+                            queue=primary_queue_dlq
+                          )
+        )
+        
+        # create a downstream SQS FIFO queue with a visibility timeout of 60 seconds
+        downstream_queue = sqs.Queue(self, "DelayFifoQueueDownstream",
+                          visibility_timeout=Duration.seconds(60),
+                          fifo=True,
+                          content_based_deduplication=True
+                          )
+        
+        #create a dynamodb table to store customer id and created timestamp
+        customer_table = dynamodb.Table(self, "CustomerTable",
+                                        table_name="DelayFifoQueueCustomerTable",
+                                        partition_key=dynamodb.Attribute(name="customer_id", type=dynamodb.AttributeType.STRING),
+                                        time_to_live_attribute="ttl"
+                                        )
+        
+        # create a lambda function to process messages from the queue
+        process_queue_function = lambda_.Function(self, "ProcessMessageLambda",
+                                                    runtime=lambda_.Runtime.PYTHON_3_9,
+                                                    code=lambda_.Code.from_asset("lambda"),
+                                                    handler="process_message.handler",
+                                                    environment={
+                                                        "QUEUE_URL": downstream_queue.queue_url,
+                                                        "TABLE_NAME": customer_table.table_name
+                                                    })
+
+        # #create an SNS topic to send notifications when primary_queue_dlq is not empty
```

Comment:
Nit: Comment format can be uniform across. e..g its "#<space>#comment" here whereas its "#<space>comment" about